pkgs/servers/sql/pgcat/default.nix

```
@@ -3,46 +3,42 @@
 , fetchFromGitHub
 , rustPlatform
 , darwin
+, nix-update-script
+, testers
+, pgcat
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "pgcat";
-  version = "1.1.1";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "postgresml";
     repo = "pgcat";
     rev = "v${version}";
```

While you're doing general fixups, why not just

```suggestion
    rev = version;
```

---

Need the `v`, since thats the tag naming convention pgcat uses.
Unless you're suggesting something else?